win32/Lib/win32timezone.py

```diff
@@ -316,7 +334,7 @@ class TIME_ZONE_INFORMATION(_SimpleStruct):
     ]
 
 
-class DYNAMIC_TIME_ZONE_INFORMATION(_SimpleStruct):
+class DYNAMIC_TIME_ZONE_INFORMATION(TIME_ZONE_INFORMATION):
```

Comment:
This is part of the fix for `__init_from_other`: `TimeZoneDefinition` needs to be a subclass of `TIME_ZONE_INFORMATION` !

Reply:
Fixed by #2339